gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -156,18 +156,19 @@ def _merge_headers(known_args, pipeline_args, pipeline_mode):
     options.view_as(pipeline_options.StandardOptions).runner = 'DirectRunner'
 
   google_cloud_options = options.view_as(pipeline_options.GoogleCloudOptions)
+  merge_headers_job_name = '-'.join([
+      _MERGE_HEADERS_JOB_NAME,
+      datetime.datetime.now().strftime('%Y%m%d-%H%M%S')])
   if google_cloud_options.job_name:
-    google_cloud_options.job_name += '-' + _MERGE_HEADERS_JOB_NAME
+    google_cloud_options.job_name += '-' + merge_headers_job_name
   else:
-    google_cloud_options.job_name = _MERGE_HEADERS_JOB_NAME
+    google_cloud_options.job_name = merge_headers_job_name
 
   temp_directory = google_cloud_options.temp_location or tempfile.mkdtemp()
   # Add a time prefix to ensure files are unique in case multiple
```

Comment:
nit: please move this above the merge_headers_job_name line.

Reply:
Thanks for the suggestion! Done.